pkgs/applications/editors/vim/plugins/vim-utils.nix

```diff
@@ -421,7 +421,7 @@ rec {
 
   # figures out which python dependencies etc. is needed for one vim package
   requiredPluginsForPackage = { start ? [], opt ? []}:
-    start ++ opt;
+    (map (p: { plugin = p; optional = false; }) start) ++ (map (p: { plugin = p; optional = true; }) opt);
```

Comment:
I am not sure you need to change this: we want to collect all the transitive dependencies. I think the test should pass if you leave it untouched.